index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta charset="UTF-8" />
+<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<title>My Portfolio</title>
+<link rel="preconnect" href="https://fonts.googleapis.com" />
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;700&display=swap"
+    rel="stylesheet" />
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous" />
+<link rel="stylesheet" href="style.css" />
+
+<body>
+    <!-- Top Navigation Menu -->
+
+    <div class="topnav">
+        <a href="#home" class="crete-font">SadiQ</a>
+        <div id="myLinks">
+            <a href="#Portofolio">Portofolio</a>
+            <a href="#About">About</a>
+            <a href="#Contact">Contact</a>
+        </div>
+        <a class="icon">
+            <i class="fa fa-bars"></i>
+        </a>
```

Comment:
- [ ] All this block of code has to be wrapped with a nav tag. This will make your code more semantic and also will add SEO support to the website.